README.md

```diff
@@ -4,11 +4,50 @@ This is the userspace API for the robot, which is what students will interface w
 
 # How to use
 
-Currently the code is designed to be imported then ran, however we hope to move the robot initialisation into the __init__.py to not need this
+Currently the code is designed to be imported then ran, however we hope to move the robot initialisation into the `__init__.py` to not need this
+
+Markers can be seen by:
+
+``` python
+markers = r.cameras[0].see()
+# or
+markers = r.camera.see()
+```
+
+Servo positions can be set by:
 
 ``` python
-from robot import Robot
-robot = Robot() # TODO move this to the __init__.py so students don't need to do it
+r.servo_boards[0].ports[0].position = -1
+# or
+r.servo_board.ports[0].position = -1
```

Comment:
it picks the first one, which sadly right now it's undefined as to which is first.